### ISSUE TITLE
Translation issues

### ISSUE THREAD

**Issue by trendschau** [typemill/typemill#197]
See: https://github.com/typemill/typemill/pull/196

**Comment by iusvar** [comment]
hi, as soon as i can i will check the problem

**Comment by iusvar** [comment]
Hi trendschau,

for the first point, just delete the colon from the fr.yaml translation file in the system/author/languages ​​folder. In practice from

```
NEXT_STEP: Prochaine étape:
GET_HELP: Obtenir de l'aide:
```

to

```
NEXT_STEP: Prochaine étape
GET_HELP: Obtenir de l'aide
```

I do it with a commit.

For the second point I have to make a premise. At this time, to get a translation, the original string in the TWIG files is capitalized and only some characters (space, period, comma and hyphen) are transformed into underscore.

Previously the colon and the open and closed round brackets were also transformed: with the latest commits, in develop not yet joined to master, I made this change.

Generally, when I make these kinds of changes, I update all the available languages, even the ones I don't know (I don't have to translate anything but just adjust the transformations).

Oliviercrouzet could not know about this situation except by going to investigate the pending commits in develop and not even you by accepting its merge.

At this point, just merge what is missing in the next version.

**Comment by trendschau** [comment]
Thank you, great!!!

**Comment by trendschau** [comment]
Published with version 1.4.0
